commands/export.go

```diff
@@ -164,19 +165,21 @@ func exportDownloadCmdF(c client.Client, command *cobra.Command, args []string)
 	}
 
 	resume, _ := command.Flags().GetBool("resume")
+	if resume {
+		printer.PrintWarning("The --resume flag has been deprecated and now the tool resumes a download automatically. The flag will be removed in a future version.")
+	}
```

Comment:
We can use:

ExportDownloadCmd.Flags().MarkHidden("resume")
ExportDownloadCmd.Flags().MarkDeprecated("resume", "This has been deprecated and now the tool resumes a download automatically. The flag will be removed in a future version.")